packages/cardano-services/package.json

```diff
@@ -29,23 +29,23 @@
     "cleanup:nm": "shx rm -rf node_modules",
     "cleanup": "run-s cleanup:dist cleanup:nm",
     "cli": "ts-node --transpile-only src/cli.ts",
-    "compose:up": "docker compose -p cardano-services-$NETWORK -f docker-compose.yml -f ../../compose/common.yml -f ../../compose/pg-agent.yml ${FILES:-} up",
+    "compose:up": "docker compose --env-file config/environments/.env.$NETWORK -p cardano-services-$NETWORK -f docker-compose.yml -f ../../compose/common.yml -f ../../compose/pg-agent.yml ${FILES:-} up",
```

Comment:
`/config` is a actually a sub-repo of https://github.com/input-output-hk/cardano-configurations, so it will be easier to source our own config in a different dir. We should probably have named it `cardano-configuration` in hindsight. Maybe `configuration`, then we could always refactor later, or if you're up for it, can do the it now.